### ISSUE TITLE
Large slowdown of TikZExternalize for each author

### ISSUE THREAD

**Issue by aquileia** [duetosymmetry/orcidlink-LaTeX-command#7]
As you're generating the same TikZ figure again and again, TikZExternalize treats each Orcid logo as a new figure and runs a separate compilation for every author, which takes a very long time. That also generates a bunch of extra files, though that's less of an issue.

I believe this should be a trivial fix, you just need to add
\tikzset{/tikz/external/export next/.try=false}
within the tikzpicture environment. This way, the logo won't use externalization.

MWE:
```
\documentclass{article}
\usepackage{orcidlink}
\usetikzlibrary{external} \tikzexternalize  % comment this out to verify that the extra command doesn't break anything

\begin{document}
    \tikzset{/tikz/external/export next/.try=false} % comment this out to see the issue
    \orcidlink{0000-0000-0000-0000}
\end{document}
```

**Comment by duetosymmetry** [comment]
Thanks, @aquileia, for the report! I tried to address this in 367ef3a, and also tried to test it with a simple example. Can you please test the version of orcidlink.sty provided in that commit and see if it works properly for you?

**Comment by duetosymmetry** [comment]
Closed by #9
